package.json

```diff
@@ -21,11 +21,42 @@
   "devDependencies": {
     "@octokit/types": "^6.17.2",
     "@types/jest": "^26.0.23",
+    "@typescript-eslint/eslint-plugin": "^4.28.1",
+    "@typescript-eslint/parser": "^4.28.1",
     "@vercel/ncc": "^0.28.6",
+    "eslint": "^7.29.0",
+    "eslint-config-haykam": "^1.10.0",
```

Comment:
What does this do? I would prefer to stick to defalts

Reply:
That's my ESLint configuration which enables [lots of opinionated rules](https://github.com/haykam821/ESLint-Config-Haykam/blob/dc44224bc9a7339f70472873f197b77733501614/eslint-config.js). It's generally in line with the existing Java conventions such as using tabs and double quotes.

Reply:
My main worry is keeping this maintained and updated, what we have now is super simple with no custom config, so nothing to worry about.

This isnt a java project so using java's conventions makes little sense to me? surely using javascripts would be a better way to go? The main point of a linter is just to ensure that contributions meet a standard to save a reviewer pointing out styling issues.

What are eslint's defaults like? Why is it better than what I setup?